po/uk.po

```diff
@@ -1723,9 +1721,9 @@ msgid "failed to parse binding"
 msgstr "не вдалося розібрати прив'язку клавіші"
 
 #: src/keymap.cpp:1028
-#, fuzzy, c-format
+#, c-format
 msgid "unknown context: %s"
-msgstr "невідома команда `%s'"
+msgstr "невідомий контекст %s"
```

Comment:
Both the English and Russian messages contain a colon to separate the message prefix and the actual context value. So, I reckon the Ukrainian translation might want to have one, too. But maybe not.

Reply:
Yep, you're totally right. Thanks for spotting that!